tests/scripts/all.sh

```diff
@@ -5114,16 +5114,15 @@ component_test_tls13_only_record_size_limit () {
 
 component_build_mingw () {
     msg "build: Windows cross build - mingw64, make (Link Library)" # ~ 30s
-    scripts/config.py unset MBEDTLS_AESNI_C # AESNI depends on cpu modifiers
-    make CC=i686-w64-mingw32-gcc AR=i686-w64-mingw32-ar LD=i686-w64-minggw32-ld CFLAGS='-Werror -Wall -Wextra' WINDOWS_BUILD=1 lib programs
+    make CC=i686-w64-mingw32-gcc AR=i686-w64-mingw32-ar LD=i686-w64-minggw32-ld CFLAGS='-Werror -Wall -Wextra -maes -msse2 -mpclmul' WINDOWS_BUILD=1 lib programs
```

Comment:
Do we need to we `make lib` without  `MBEDTLS_AESNI_C` to check that it works on mingw?

Reply:
Yes, but I mean to check that the C code builds under mingw, which is quite a "strange" environment - just `lib`, not `programs`, which shouldn't take long

Reply:
Sorry, I didn't get the point. We do build `lib` and `programs` and later `tests` for both static and dynamic linked libraries in this `mingw` test...